JenkinsfileBuildTrigger

```diff
@@ -0,0 +1,84 @@
+#!groovy
+
+
+def libraries = ['ci-pipeline':              ['master', 'https://github.com/CentOS-PaaS-SIG/ci-pipeline.git'],
+                 'upstream-fedora-pipeline': ['master', 'https://github.com/CentOS-PaaS-SIG/upstream-fedora-pipeline.git']]
+
+libraries.each { name, repo ->
+    library identifier: "${name}@${repo[0]}",
+            retriever: modernSCM([$class: 'GitSCMSource',
+                                  remote: repo[1]])
+
+}
+
+properties(
+        [
+                buildDiscarder(logRotator(artifactDaysToKeepStr: '', artifactNumToKeepStr: '500', daysToKeepStr: '', numToKeepStr: '500')),
+                parameters(
+                        [
+                                string(description: 'Red Hat UMB Msg', defaultValue: '{}', name: 'CI_MESSAGE')
+                        ]
+                ),
+                pipelineTriggers(
+                        [
+                                [$class: 'CIBuildTrigger', checks: [[new: '1']], providerName: 'fedora-fedmsg', selector: 'topic = "org.fedoraproject.prod.buildsys.build.state.change"']]
+                )
+        ]
+)
+
+def TRIGGER_RETRY_COUNT = 3
+def stepName = null
+def targetRelease = null
+
+node() {
+
+    ciMetrics.prefix = 'upstream-fedora-pipeline'
+    packagepipelineUtils.cimetrics = ciMetrics
+
+    try {
+        packagepipelineUtils.ciPipeline {
+            stepName = 'upstream-fedora-pipeline-build-trigger'
+            stage(stepName) {
+
+                packagepipelineUtils.timedPipelineStep(stepName: stepName, debug: true) {
+
+                    print "CI_MESSAGE"
+                    print CI_MESSAGE
+                    pipelineUtils.flattenJSON(env.CI_MESSAGE)
+                    pipelineUtils.repoFromRequest(env.msg_request)
+                    pipelineUtils.updateBuildDisplayAndDescription()
+                    targetRelease = packagepipelineUtils.checkRelease(env.msg_release)
+
+                }
+
+            }
+
+            if (targetRelease) {
+                stepName = 'schedule build'
+                buildJob = "fedora-${targetRelease}-build-pipeline"
+                stage(stepName) {
+
+                    retry(TRIGGER_RETRY_COUNT) {
+                        packagepipelineUtils.timedPipelineStep(stepName: stepName, debug: true) {
+
+                            build job: buildJob,
+                                    parameters: [string(name: 'PROVIDED_KOJI_TASKID', value: env.fed_msg_task_id),
+                                                 string(name: 'ARTIFACT_TYPE', value: buildJob)],
```

Comment:
We have no storage behind the all package pipelines so there is no ARTIFACT_TYPE. Everything is just in workspace